maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -275,7 +275,7 @@ class Orchestra(
     /**
      * Returns true if the command mutated device state (i.e. interacted with the device), false otherwise.
      */
-    private fun executeCommand(maestroCommand: MaestroCommand, config: MaestroConfig?): Boolean {
+    private fun executeCommand(maestroCommand: MaestroCommand, config: MaestroConfig?, originalCommand: MaestroCommand): Boolean {
```

Comment:
What is difference between maestro command and originalCommand. Can't we use maestroCommand here?